lms/djangoapps/onboarding/choices.py

```diff
@@ -0,0 +1,50 @@
+FUNCTIONS_LABELS = (
```

Comment:
These are basically constants. How about we place them inside constants file instead of creating a new one?

Reply:
I thought the same initially. However, @saad-waqar advised to create a separate file for choices, because altering choices directly alters the models while this is not the case with constants, and that is why he wanted choices to be stored in a separate file.

Reply:
aright.